.github/workflows/nightly-check-db.yml

```diff
@@ -0,0 +1,43 @@
+name: Nightly check db
+
+on: [push]
+  # schedule:
+  #   - cron: '0 3 * * *'
+
+env:
+  PYTHONPATH: ${{ github.workspace }}
+
+jobs:
+  download-snapshaot-move-data-check-db:
+
+    runs-on: ubuntu-latest
+    container: quarkchaindocker/pyquarkchain:mainnet1.1.1
+    
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Download db snapshot
+      working-directory: /temp
```

Comment:
can combine downloading and moving data to a single step

Reply:
ok~

Reply:
Done.